### DISCUSSION TITLE
Bluetooth loss when on phone with a headset

### DISCUSSION THREAD

**Discussion by AnnaJadot** [NightscoutFoundation/xDrip#2992] [General]
Hi

I use Xdrip+ with Dexcom One.

My phone is also connected to a headset via Bluetooth.

When I'm on a phone call using the headset I loose the connection with Dexcom.

Is there a way to keep them both connected when I'm on the line?
Yesterday I spend 2 hours on the line and when I hung up, I've noticed i was very low.

Is there a setting?

Thanks

**Comment by rudysroost** [comment]
I've been experiencing the same problem with both G6 and G7. Although I've managed to cope with it, it would be wonderful if a solution could be found.
